db/migrate/20150328164617_devise_create_users.rb

```diff
@@ -19,40 +19,28 @@ def change
       t.inet     :current_sign_in_ip
       t.inet     :last_sign_in_ip
 
-      ## Confirmable
-      # t.string   :confirmation_token
-      # t.datetime :confirmed_at
-      # t.datetime :confirmation_sent_at
-      # t.string   :unconfirmed_email # Only if using reconfirmable
+      t.string  :first_name,                 null: false
+      t.string  :last_name,                  null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.